pages/common/babel.md

```diff
@@ -0,0 +1,35 @@
+# babel
+
+> A compiler for writing next generation JavaScript.
```

Comment:
Shouldn't it be transpile ?

Reply:
Yes, I think.

Reply:
Yes. From [here](https://www.stevefenton.co.uk/2012/11/compiling-vs-transpiling/):

> Compiling
>  - is the general term for taking source code written in one language and transforming into another
> 
> Transpiling
>  - is a specific term for taking source code written in one language and transforming into another language that has a similar level of abstraction